osu.Game/Scoring/Legacy/ScoreInfoExtensions.cs

```diff
@@ -3,13 +3,58 @@
 
 #nullable disable
 
+using System;
 using System.Collections.Generic;
+using System.Linq;
 using osu.Game.Rulesets.Scoring;
 
 namespace osu.Game.Scoring.Legacy
 {
     public static class ScoreInfoExtensions
     {
+        public static long GetDisplayScore(this ScoreProcessor scoreProcessor, ScoringMode mode)
+            => getDisplayScore(scoreProcessor.Ruleset.RulesetInfo.OnlineID, scoreProcessor.TotalScore.Value, mode, scoreProcessor.MaximumStatistics);
+
+        public static long GetDisplayScore(this ScoreInfo scoreInfo, ScoringMode mode)
+            => getDisplayScore(scoreInfo.Ruleset.OnlineID, scoreInfo.TotalScore, mode, scoreInfo.MaximumStatistics);
+
+        private static long getDisplayScore(int rulesetId, long score, ScoringMode mode, IReadOnlyDictionary<HitResult, int> maximumStatistics)
+        {
+            if (mode == ScoringMode.Standardised)
+                return score;
+
+            double multiplier;
+
+            switch (rulesetId)
+            {
+                case 0:
+                    multiplier = 36;
+                    break;
+
+                case 1:
+                    multiplier = 22;
+                    break;
+
+                case 2:
+                    multiplier = 28;
+                    break;
+
+                case 3:
+                    multiplier = 16;
+                    break;
+
+                default:
+                    return score;
+            }
```

Comment:
I'd like to see this split out into its own method with xmldoc explaining what the multipliers are used for and (very roughly) how they were obtained.

Right now there's no documentation explaining this.